Modules/documents/Controller/DocumentslistController.php

```diff
@@ -63,28 +63,26 @@ public function indexAction($id_space) {
     public function editAction($id_space, $id) {
         $this->checkAuthorizationMenuSpace("documents", $id_space, $_SESSION["id_user"]);
         $lang = $this->getLanguage();
-
         $model = new Document();
         $data = $model->get($id_space, $id);
 
         $form = new Form($this->request, "DocumentsEditAction");
         $form->setTitle(DocumentsTranslator::Edit_Document($lang));
         $form->addText("title", DocumentsTranslator::Title($lang), true, $data["title"]);
         $form->addUpload("file_url", DocumentsTranslator::File($lang));
-        $form->setValidationButton(CoreTranslator::Save($lang), "documentsedit/" . $id_space);
+        $form->setValidationButton(CoreTranslator::Save($lang), "documentsedit/" . $id_space . "/" . $id);
```

Comment:
why ? once edited we should go back to listing

Reply:
no, we pass through editAction a second time, going into if ($form->check()) block, then are being redirected to list with $this->redirect("documents/" . $id_space) line 88. But now we have the document id (with this line, 73) and can tell if it's a new document or an existing one